### ISSUE TITLE
letterSpacing not working

### ISSUE THREAD

**Issue by feri-irawan** [Automattic/node-canvas#2181]
# Code
```js
  const canvas = createCanvas(width, height);
  const ctx = canvas.getContext("2d");
  
  ctx.font = `bold ${fontSize}px customFont`;
  ctx.textAlign = "center";
  ctx.fillStyle = "#fff";
  ctx.letterSpacing = "29px"; // <------
  ctx.strokeStyle = "red";
  ctx.lineWidth = 5;
```

**Comment by zbjornson** [comment]
Duplicate of #1014. It's part of the Canvas spec now, so nothing blocking its implementation (the old discussion there was blocked on how to add it without adding a non-standard feature).
